data/transactions/logic/assembler_test.go

```diff
@@ -397,7 +397,8 @@ type expect struct {
 	s string
 }
 
-func testMatch(t *testing.T, actual, expected string) {
+func testMatch(t testing.TB, actual, expected string) {
```

Comment:
why switching to testing.TB? I think this breaks the test cases when method called with t that's of type *testing.T

Reply:
This allows *testing.T and *testing.B, which is used for benchmarking.  By doing this I can call testProg() which uses testMatch() from a Test or a Benchmark.